backend/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+const env = require('./dev/env')
+const db = require('./src/config/db')
+
+const port = env.PORT
+
+app.use(bodyParser.json())
+
+app.post('/api/v1/games', (req, res) => {
+  const playerId = req.body.playerId
+  const token = req.body.token
```

Comment:
1. to create a game, just the rows and columns are sent in the body of the request, the player Id and the token should be generated in the backend.
2. The "Game" module is missing, you should not save the game directly in the database in this layer of the app